runtime/layers/fpm-dev/Dockerfile

```diff
@@ -13,9 +13,8 @@ USER root
 COPY --from=build_extensions /tmp/*.so /tmp/
 RUN cp /tmp/*.so $(php -r "echo ini_get('extension_dir');")
 
-FROM bref/php-${PHP_VERSION}-fpm
+FROM bref/php-${PHP_VERSION}-fpm as without_extensions
```

Comment:
Will help for other new php versions upgrade as most of the time blackfire or xdebug are not ready during beta releases.